scipy/special/cephes/erfinv.c

```diff
@@ -17,6 +17,17 @@ double erfinv(double y) {
     const double domain_lb = -1;
     const double domain_ub = 1;
 
+    const double thresh = 1e-7;
```

Comment:
```
import scipy.special
import mpmath
import math

mpmath.dps = 50

M_2_SQRTPI = 2 / math.sqrt(math.pi)

for i in range(1, 20):
    y = 10 ** -i
    correct = float(mpmath.erfinv(y))

    s = scipy.special.erfinv(y)
    s2 = (y / M_2_SQRTPI)

    print (f"{i:2} {(s - correct) / correct:.2e} {(s2 - correct) / correct:.2e}")
```

```
 1 1.09e-15 -2.63e-03
 2 1.17e-15 -2.62e-05
 3 -1.10e-13 -2.62e-07
 4 -1.10e-13 -2.62e-09
 5 6.55e-12 -2.62e-11
 6 -8.23e-11 -2.62e-13
 7 5.84e-10 -2.54e-15
 8 -6.08e-09 0.00e+00
 9 8.27e-08 0.00e+00
...
19 -1.00e+00 0.00e+00
```

I think the single term taylor series generates better results starting from `y < 6e-6`